TS1036 error for v4

I get a `TS1036: Statements are not allowed in ambient contexts.` error when importing from the package and compiling with `tsc`.

**Steps to reproduce:**
1. Open https://stackblitz.com/edit/node-npm-run-script-bbegdr?file=package.json
2. Wait for `npm install` to complete
3. Run `npm run build`
4. Observe the error

```
> typescript@0.0.0 build
> tsc

node_modules/xmllint-wasm/index-node.d.ts:6:2 - error TS1036: Statements are not allowed in ambient contexts.

6 };
   ~


Found 1 error in node_modules/xmllint-wasm/index-node.d.ts:6
```

I'm able to reproduce the error locally as well, but I shared the Stackblitz link because it's easier to share and reproduce.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "xmllint-wasm",
-  "version": "4.0.0",
+  "version": "4.0.1",
   "description": "Fork of https://www.npmjs.com/package/xmllint with some updates",
   "main": "index-node.js",
   "browser": "index-browser.mjs",

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -3,11 +3,11 @@ export type XMLString = string;
 export interface XMLFileInfo {
   readonly fileName: string;
   readonly contents: XMLString;
-};
+}
 
 export type XMLInput = XMLString | XMLFileInfo;
 
-interface Schema { readonly schema: XMLInput | ReadonlyArray<XMLInput> };
+interface Schema { readonly schema: XMLInput | ReadonlyArray<XMLInput> }
 interface Normalization {
   /**
    * Pass either --format or --c14n to xmllint to get a formatted
@@ -16,7 +16,7 @@ interface Normalization {
    * normalization: 'c14n' performs W3C XML Canonicalisation (C14N).
    */
   readonly normalization: 'format' | 'c14n';
-};
+}
 
 interface XMLLintOptionsBase {
   /**
@@ -68,7 +68,7 @@ export interface XMLValidationError {
     readonly fileName: string;
     readonly lineNumber: number;
   };
-};
+}
 
 export interface XMLValidationResult {
   readonly valid: boolean;
@@ -106,6 +106,6 @@ interface MemoryPagesConstant {
 	 * The maximum possible value for the memory options (4GiB).
 	 */
 	readonly max: number;
-};
+}
 
 export const memoryPages: MemoryPagesConstant;

--- a/typings-test/tsconfig.json
+++ b/typings-test/tsconfig.json
@@ -5,7 +5,7 @@
     "rootDir": "../",
     "noEmit": true,
     "strict": true,
-    "skipLibCheck": true,
+    "skipLibCheck": false,
 		"noImplicitAny": true
   },
 	"include": [

